### DISCUSSION TITLE
request failed with status code 500 这个是怎么回事

### DISCUSSION THREAD

**Discussion by MediocreYYYY** [dreamhunter2333/cloudflare_temp_email#403] [General]
![image](https://github.com/dreamhunter2333/cloudflare_temp_email/assets/88574095/eb0fcd85-c120-4227-a01a-d0abdbe1103e)

**Comment by dreamhunter2333** [comment]
可以看下控制台的具体报错么

**Comment by naodeng** [comment]
我这边是这个错误：vendor-bd5bJ8Bc.js:3542 
        
        
       GET https://cloudflare_temp_email.naodeng.workers.dev/api/new_address?name=12345&domain=naodeng.com.cn 500 (Internal Server Error)

**Comment by naodeng** [comment]
https://production.email-8w0.pages.dev/ 我的地址 另外为啥还有 login 按钮

**Comment by dreamhunter2333** [comment]
login 是新加的，之前在顶栏，我试了下看起来没问题

![image](https://github.com/dreamhunter2333/cloudflare_temp_email/assets/32295532/ba731176-90e9-4cf5-838f-cb84674b2d8c)


> https://production.email-8w0.pages.dev/ 我的地址 另外为啥还有 login 按钮
